src/application-manager.coffee

```diff
@@ -68,11 +68,11 @@ createApplicationManagerRouter = (applications) ->
 
 	return router
 
-module.exports = class ApplicationManager extends EventEmitter
+exports.ApplicationManager = class ApplicationManager extends EventEmitter
 	constructor: ({ @logger, @config, @db, @eventTracker, @deviceState }) ->
 		@docker = new Docker()
 		@images = new Images({ @docker, @logger, @db, @config })
-		@services = new ServiceManager({ @docker, @logger, @images, @config })
+		@services = new ServiceManager({ @docker, @logger, @config })
```

Comment:
There were a load of examples like this where we pass unexpected/unused parameters that I removed